From c7834f4aaba0118a827b8d60a931bec84009d0e0 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Thu, 26 Apr 2018 15:12:04 +0800
Subject: [PATCH 2/2] driver: usb: common: correct misleading indentation

drivers/usb/common/common.c:150:2: warning: this 'for' clause does not guard... [-Wmisleading-indentation]
  for (i = 0; i < ARRAY_SIZE(speed_names); i++)
  ^~~
drivers/usb/common/common.c:154:3: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the 'for'
   return USB_SPEED_UNKNOWN;
   ^~~~~~

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/usb/common/common.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/common/common.c b/drivers/usb/common/common.c
index 176dee0..f4165f2 100644
--- a/drivers/usb/common/common.c
+++ b/drivers/usb/common/common.c
@@ -151,7 +151,7 @@ enum usb_device_speed of_usb_get_maximum_speed(struct device_node *np)
 		if (strcmp(maximum_speed, speed_names[i]) == 0)
 			return i;
 
-		return USB_SPEED_UNKNOWN;
+	return USB_SPEED_UNKNOWN;
 }
 EXPORT_SYMBOL_GPL(of_usb_get_maximum_speed);
 
-- 
1.8.5.2.233.g932f7e4

